modules/sc-mesh-secure-deployment/src/nats/scripts/cli_radio_on.py

```diff
@@ -8,7 +8,7 @@ async def main():
     # Connect to NATS!
     nc = await client.connect_nats()
 
-    cmd_dict = {"api_version": 1, "cmd": "UP"}
+    cmd_dict = {"api_version": 1, "cmd": "UP", "radio_index": "0"}
```

Comment:
Above I can guess from the script names (...hostapd.py, ...wpa.py, etc) that "radio_index" only applies to WiFi interfaces but what about here: would "cli_radio_on.py" be called to turn OSF on for example? :)

Reply:
OSF is not yet part of multiradio.  It was agreed when this work was ongoing.

Reply:
Maybe I did not express myself very well but ok if I hopefully understand correctly now that OSF will be added to this in the future then the name does not need to be WiFi related 🙂